### PR TITLE
[DONT MERGRE] Release/13.0.x

### PR DESCRIPTION
**Pull request by kderme**

### CODE REVIEW

File: doc/configuration.md [hunk 1]
```diff
@@ -4,11 +4,26 @@ The initial design of db-sync was an one fit all approach, where it would serve
 
 ### --disable-ledger
 
-One of the features of db-sync that uses the most recourses is that it maintains a ledger state and it replays all the ledger rules. This is the only way to get historic rewards an other things that are not included in the blocks, like historic stake distribution, ada pots, epoch params etc. The flag --disable-ledger provides the option to turn off these feauture an significantly reduces usage of memory (by up to 10GB) and sync time.
```
Comment (erikd) [inline doc/configuration.md:7]: The changelog update should be on this release branch, but the documentation change should be on master.

**Comment by erikd** [comment]
@kderme I pulled the documentation changes into master, rebased this release branch and force pushed.

**Comment by erikd** [comment]
@kderme Even after I fixed the compile errors, this still failed about 10 of the tests.

Also, the dependency update should be a separate PR which is merged to master and then cherry picked back onto this release branch.

**Comment by kderme** [comment]
I wanted to pass them through ci before cutting the tag. Will cherry-pick to a different pr.

**Comment by erikd** [comment]
Closing this as old/redundant.

**Pull request closed without merging**